src/lib.rs

```diff
@@ -9,18 +9,25 @@ mod findex;
 mod ovec;
 mod secret;
 mod symmetric_key;
+#[cfg(any(test, feature = "bench", feature = "test-utils"))]
+mod test;
 mod value;
 
 pub use address::Address;
 pub use adt::{IndexADT, MemoryADT};
+pub use encoding::{dummy_decode, dummy_encode};
```

Comment:
I am not sure we want to publicly expose these, as they are really rudimentary...

Reply:
exposing these was the original goal of this PR as there is no clear way of instantiation in cloudproof rust without them